src/app/catalog/components/issuer-catalog/issuer-catalog.component.ts

```diff
@@ -428,12 +419,39 @@ export class IssuerCatalogComponent extends BaseRoutableComponent implements OnI
 	}
 
 	openMap() {
-		this.badgesDisplay = "map";
+		this.badgesDisplay = 'map';
 		let that = this;
 		setTimeout(function () {
 			that.mapObject.resize();
 		}, 10);
 	}
+
+	prepareTexts(){
+		this.issuerKeys = {
+			schule: 'Schulen',
+			hochschule: 'Hochschulen und Universitäten',
+			andere: 'Andere (Bibliotheken, Museen, FabLabs, Unternehmen, Vereine, ...)',
+			'n/a': 'Keine Angabe',
+		};
+	
+		this.plural = {
+			issuer: {
+				'=0': this.translate.instant('Issuer.noInstitutions'),
+				'=1': '1 Institution',
```

Comment:
Shouldn't this also be part of the translations, just in case we at some point have a language where it doesn't coincide?

Reply:
Some words are the same in German and English. But I think I'll update it to make it easy for adding new languages in the future.